go-controller/pkg/ovn/obj_retry.go

```diff
@@ -957,15 +957,6 @@ func (oc *Controller) deleteResource(objectsToRetry *RetryObjs, obj, cachedObj i
 			return extraParameters.gp.delNamespaceAddressSet(namespace.Name)
 		})
 
-	case factory.PeerPodForNamespaceAndPodSelectorType:
```

Comment:
@cathy-zhou hmm, the case that is still there is;

```
	case factory.PeerPodForNamespaceAndPodSelectorType:
		extraParameters := objectsToRetry.extraParameters.(*NetworkPolicyExtraParameters)
		return oc.handlePeerPodSelectorDelete(extraParameters.gp, obj)
```

I *think* that's the right one to keep because this is the peer pod selector type, while the one you're deleting is cleaning up the local pod stuff. But would be good to get another review...

@trozet @jcaamano @tssurya ?

Reply:
It looks ok to me, based that we are actually removing dead code.

Reply:
yeah we should delete that, I have the same change as  part of another PR

Reply:
OK, thanks for the confirmation.